src/components/Sum.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const Calc = ({ a = 0, b = 0 }) => (
+  <p>
+    {`Sum of ${a} and ${b} is ${a + b}`}
+  </p>
+);
```

Comment:
Though it's optional task, but I believe you can redo this component with using both PropTypes and defaultProps